tests/sentry/utils/performance_issues/test_http_overhead_detector.py

```diff
@@ -260,3 +260,23 @@ def test_detect_other_location(self):
                 evidence_display=[],
             )
         ]
+
+    def test_none_request_start(self):
+        url = "https://example.com/api/endpoint/123"
+        event = _valid_http_overhead_event("/api/endpoint/123")
+
+        # Include an invalid span to ensure it's not processed
+        span = create_span(
+            "http.client",
+            desc=url,
+            duration=1000,
+            data={
+                "url": url,
+                "network.protocol.version": "1.1",
+                # request_start is not present
```

Comment:
AFAICT the erroring case here is when `request_start` is actually present, but with the value `None`. (If it wasn't present, `span_data.get("http.request.request_start", 0)` would evaluate to the given default value `0`).

```suggestion
                "request_start": None,
```

Reply:
Oh yes good catch, thanks!